core/src/main/java/nl/nn/adapterframework/http/rest/ApiListener.java

```diff
@@ -286,6 +288,22 @@ public String getOperationId() {
 		return operationId;
 	}
 
+	@IbisDoc({"12", "Comma separated list of parameter names defined in the header. Do not use followings 'Accept', 'Content-Type' or 'Authorization'", ""})
```

Comment:
I think 'defined in the header' should be: 'passed as http header'.
Why can the specified list not be used?